### DISCUSSION TITLE
[50] schemeorg front end editing

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#46991] [3. Approved Features]
### Steps to reproduce the issue
try to edit or creat an article in the front end

### Expected result
Can select and set the values for schema

### Actual result
No option to do anything for schema

# Approval Information
Approved: 15.07.2026
Minimum Target Version: 6.2

**Comment by brianteeman** [comment]
Is this by design or an oversight?

**Comment by HLeithner** [comment]
I would expect not yet implemented. @bembelimen

**Comment by drmenzelit** [comment]
**The Joomla Maintainer Team decided at the 15.07.2026 meeting to accept this feature.**
